robotidy/transformers/NormalizeTags.py

```diff
@@ -0,0 +1,77 @@
+from robot.api.parsing import ModelTransformer, Tags, Token
+import click
+
+class NormalizeTags(ModelTransformer):
+    """
+    Normalize tag names by normalizing case and removing duplicates.
+    example usage:
+
+        robotidy --transform NormalizeTags:case=lowercase test.robot
+
+    Other supported cases: uppercase, titlecase. The default is lowercase.
+
+    You can also run it to remove duplicates but preserve current case using ``only_remove_duplicates`` parameter:
+
+        robotidy --transform NormalizeTags:only_remove_duplicates=True test.robot
+
+    See https://robotidy.readthedocs.io/en/latest/transformers/NormalizeTags.html for more examples.
+    """
+    CASE_FUNCTIONS = {'lowercase': str.lower, 'uppercase': str.upper, 'titlecase': str.title}
+
+    def __init__(self, case: str = 'lowercase', only_remove_duplicates: bool = False):
```

Comment:
instead of `only_remove_duplicates = False` we could use `normalize_case = True` - it would be then clear what transformer is doing/not doing when we set this parameter to opposite value.

Reply:
changed